config/default/kustomization.yaml

```diff
@@ -0,0 +1,73 @@
+# Adds namespace to all resources.
+namespace: opendatahub
+# Value of this field is prepended to the
+# names of all resources, e.g. a deployment named
+# "wordpress" becomes "alices-wordpress".
+# Note that it should also match with the prefix (text before '-') of the namespace
+# field above.
+namePrefix: ""
```

Comment:
We should set a name prefix, if not the resulting objects will have a generic name that can overwrite other existing objects in the cluster (cluster roles, bindings, etc).

For example, without the prefix the operator deployment will be named as `controller-manager`. I suggest setting it to `odh` and the deployment will be named as `odh-controller-manager`.

Reply:
Yes I agree. I missed the generic name of the objects

Reply:
Fixed